src/main/java/totalcross/sample/TCSampleApplication.java

```diff
@@ -6,6 +6,10 @@
 public class TCSampleApplication {
 	public static void main(String[] args) {
 	    TotalCrossApplication.run(
-	        TCSample.class, "/r", "<PLACE YOUR KEY>", "/scr", "iphone");
+				TCSample.class,"/scr", "iphone"
```

Comment:
This file beforehand was in a code-style of 2-space indentation, but the new-lines here uses 1-tab indentation.

I think that the project should have a single and coherent code-style